src/openloco/platform/platform.posix.cpp

```diff
@@ -13,6 +13,12 @@
 #include <unistd.h>
 #endif
 
+#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__DragonFly__)
+#include <sys/sysctl.h>
```

Comment:
Might've been more useful to use c++17's `has_include` to automatically support any other systems that would require those

Reply:
FreeBSD have default compilet clang. I use clang compiler.

Reply:
but this changes are correct

Reply:
I did not say they are incorrect, just that they could be improved still.